### ISSUE TITLE
Two issues in Chapter 11

### ISSUE THREAD

**Issue by andrewsun90** [ageron/handson-ml#228]
Hi @ageron,

(1) To apply the dropout technique to the MNIST dataset, shall we set `training` to `True` in `feed_dict` (as in the 8th line of the 101th code block in https://github.com/ageron/handson-ml/blob/master/11_deep_learning.ipynb)?

(2) As you expected, `tf.nn.selu()` has already been included in the newest Tensorflow version, so probably there is no need to define a SELU function (as in the 28th code block) anymore.
Please find more information at: https://www.tensorflow.org/api_docs/python/tf/nn/selu

**Comment by ageron** [comment]
Thanks @andrewsun90 , I appreciate your feedback.
The book does say to set `training=True` when training (in the dropout section) but I somehow forgot to actually add this to the notebook, oops.  :/
Regarding SELU, I updated the comments to point to `tf.nn.selu()` (and `tf.contrib.nn.alpha_dropout()`), but I left the current use of a "home-made" `selu()` function just in case some users are still using a TF version < 1.4.  I'll eventually drop support for older TF versions.
Thanks again!
